dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryShimTest.java

```diff
@@ -58,21 +57,29 @@ public class TelemetryShimTest extends FATServletClient {
 
     @BeforeClass
     public static void setup() throws Exception {
-        if (RepeatTestFilter.isRepeatActionActive(MicroProfileActions.MP61_ID)) {
```

Comment:
My only suggestion would be adding a comment like 

//Add the correct version of the shim library for each version of OTel.

But I will not block the merge for that.